pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -533,6 +533,8 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// pull images
 				rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams/layers").RuleOrDie(),
+				// read signatures
+				rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreamimages").RuleOrDie(),
```

Comment:
this role is intended to grant permissions to call the registry endpoints. the registry is already authorizing getting signatures based on `get imagestreams/layers`, have the registry's signature endpoint get signatures from the API using the registry's client, not the user's